ext/pgsql/tests/pg_select_001.phpt

```diff
@@ -31,6 +31,11 @@ var_dump(pg_select($conn, 'phptests.bar', array('id' => 1)));
 /* Existent column */
 var_dump(pg_select($conn, 'phptests.bar', array('id4' => 4)));
 
+/* Empty array */
+var_dump(pg_select($conn, 'phptests.bar', array()));
+
+/* No array */
+var_dump(pg_select($conn, 'phptests.bar'));
```

Comment:
You might need to update the [pgsql's stub](https://github.com/php/php-src/blob/02865b1f568d21f427df83954dc78c24ca639c54/ext/pgsql/pgsql.stub.php#L964).